components/style/values.rs

```diff
@@ -2156,4 +2157,20 @@ pub mod computed {
     pub type Length = Au;
     pub type Number = CSSFloat;
     pub type Opacity = CSSFloat;
+
+    pub fn convert_rgba_to_nscolor(rgba: &RGBA) -> u32 {
```

Comment:
Sorry, I meant that this should go in ports/geckolib/values.rs (which I added in #10938). We shouldn't be doing anything with nscolor in components/style.